src/plugins/enzymes/src/FindEnzymesTask.cpp

```diff
@@ -257,9 +257,10 @@ void FindSingleEnzymeTask::prepare() {
     SequenceDbiWalkerConfig sequenceWalkerConfig;
     sequenceWalkerConfig.seqRef = sequenceObjectRef;
     sequenceWalkerConfig.range = region;
-    sequenceWalkerConfig.chunkSize = qMax(enzyme->seq.size(), chunkSize);
+    int enzymeFullLength = enzyme->getFullLength();
```

Comment:
Enzyme length is not the enzyme only, but leading and trailing N bases. We had no problems with it previously, because we had separate handling of the enzyme in `FindEnzymesAlgorithm.h`, but now we need to ad a propper number af bases to the end of sequence.